api/service/v1alpha1/service.proto

```diff
@@ -32,6 +32,7 @@ service KargoService {
   rpc PromoteStage(PromoteStageRequest) returns (PromoteStageResponse);
   rpc PromoteSubscribers(PromoteSubscribersRequest) returns (PromoteSubscribersResponse);
   rpc RefreshStage(RefreshStageRequest) returns (RefreshStageResponse);
+  rpc RequestStageVerification(RequestStageVerificationRequest) returns (RequestStageVerificationResponse);
```

Comment:
Two things here.

1. I don't think I would count verification specifically as a Stage-oriented operation. It's really verification of a piece of Freight _with respect to_ a Stage. (Side note: I've been meaning for a long time to rename `PromoteStage` for the same reason. I'll finally make an issue for it at least so we don't lose track.)

2. Much more of a nit: I suppose it's _accurate_ that this endpoints _requests_ re-verification, but the corresponding message types, `RequestStageVerificationRequest` and `RequestStageVerificationResponse` sound awkward to me.

In light of both of the above, could I suggest tentatively renaming this to `Reverify`, with corresponding message types as `ReverifyRequest` and `ReverifyResponse`?